Documentation/PageTsconfig/TceForm.rst

```diff
@@ -293,13 +293,13 @@ config
    This property is available for various levels:
 
    table level, example:
-      `TCEFORM.tt_content.header.config.max`
+      `TCEFORM.tt_content.header.config.maxitems`
```

Comment:
```suggestion
      `TCEFORM.tt_content.header.config.max`
```

I agree with @brotkrueml , thi must be left unchanges. input has the property 'max' , see above